app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm.rb

```diff
@@ -1,2 +1,20 @@
 class ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm < ManageIQ::Providers::CloudManager::Vm
+  # https://cloud.ibm.com/apidocs/vpc#get-instance
+  POWER_STATES = {
+    "running"       => "on",
+    "failed"        => "off",
+    "paused"        => "suspended",
+    "pausing"       => "suspended",
+    "pending"       => "suspended",
+    "restarting"    => "powering_up",
```

Comment:
Rubocop is complaining because you have too many spaces here.  You want to align the `=>` with the longest key so

```
POWER_STATES = {
  "running"    => ...
  "restarting" => ...
}
```